example_sns_init.yaml

```diff
@@ -25,104 +31,307 @@ description: >
     Built on chainkey cryptography, secure and true,
     An app that transcends, all we once knew...
 
-Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+# The structure allows for mapping identities to human-readable names.
+# This is currently a placeholder field and must be left empty for now.
+Principals: []
 
+# Path to the SNS Project logo on the local filesystem. The path is relative
+# to the configuration file location, unless an absolute path is given.
+# Must have less than 341,334 characters, roughly 256 Kb. The only supported
+# format is PNG.
 logo: logo.png
+
+# URL to the dapp controlled by the SNS project.
+# Must be a string of max length = 512.
 url: https://forum.dfinity.org/thread-where-this-sns-is-discussed
 
+# Configuration of the NNS proposal metadata and is shown only in
+# the NNS proposal.
+NnsProposal:
+    # The title of the NNS proposal. Must be a string of max 256 bytes.
+    title: "NNS Proposal to create an SNS named 'Rock Out'"
+
+    # The HTTPS address of additional content required to evaluate the NNS
+    # proposal. Must be a string of max 2,000 bytes.
```

Comment:
Does it have to be an HTTPS address? Does it need to be forum.dfinity.org? Above we simply talk about a "URL" of the project.

Reply:
I think we do enforce some constrains , but let's maybe check which

Reply:
We enforce https:// but not the 2000 bytes. Edited.

See https://sourcegraph.com/github.com/dfinity/ic@7b0a6b32519aada63cb05925d7459f209b7d04f5/-/blob/rs/nervous_system/common/src/lib.rs?L714